doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
Unrelated to this PR, but is this a recent thing? I remember having to use https://www.npmjs.com/package/epipebomb in the past to prevent rare, but real issues from occurring.